packages/python/accern_data/util.py

```diff
@@ -39,8 +39,26 @@ def is_test() -> int:
     return IS_TEST
 
 
+def check_filters(
+        record: Dict[str, Any], filters: Dict[str, Optional[str]]) -> bool:
+    for key, value in filters.items():
+        if field_transformation(record[key]) != value:
```

Comment:
can't use != iif the field is None

Reply:
why do you allow None anyway? would be easier if you just filter that when parsing / updating